packages/vitest/src/runtime/inspector.ts

```diff
@@ -20,8 +22,23 @@ export function setupInspect(config: ResolvedConfig) {
     if (!isOpen) {
       inspector.open()
 
-      if (config.inspectBrk)
+      if (config.inspectBrk) {
+        const firstTestFile = ctx.files[0]
+
+        // Stop at first test file
+        if (firstTestFile) {
+          session = new inspector.Session()
+          session.connect()
+
+          session.post('Debugger.enable')
+          session.post('Debugger.setBreakpointByUrl', {
+            lineNumber: 1,
+            url: new URL(firstTestFile, import.meta.url).href,
```

Comment:
This works but the debugger is not following source maps.
![image](https://github.com/vitest-dev/vitest/assets/14806298/5c69db64-f6a8-4a25-b5ce-07f65d46092b)

However when using `debugger` statement in test, dev tools show original source nicely:
![image](https://github.com/vitest-dev/vitest/assets/14806298/5a445b02-6957-4854-85ad-5c04072477aa)

I'll need to look more into the documentation: https://chromedevtools.github.io/devtools-protocol/v8/Debugger/#method-setBreakpointByUrl

Reply:
Oh right the first line is not on source maps. If breakpoint is set on line 3, then source maps are followed correctly. 
![image](https://github.com/vitest-dev/vitest/assets/14806298/87b5373b-8cb9-4615-a619-044a340362f0)

This requires a bit similar knowledge of vite-node's wrapper code like V8 code coverage needs. First line contains `'use strict'`, second one the execution wrapper, and then there's the code.

Reply:
Resolved in 945a23f by making sure vite-node's source maps always contain mappings for the first line.